test/travelagent-test.js

```diff
@@ -113,7 +114,7 @@ describe('Travel Agent', function() {
 
     beforeEach(() => {
     user = new User(userData);
```

Comment:
does user hold all data?

Reply:
user just holds the password and whether they are logged in. I might change it later, it does seem kind of redundant.